tunnel_routes_test.go

```diff
@@ -0,0 +1,205 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//List Tunnel Routes (GET accounts/:account_identifier/teamnet/routes)
+func TestAPI_ListTunnelRoutes(t *testing.T) {
+	setup(UsingAccount(testAccountID))
```

Comment:
we don't need the call `UsingAccount` here, we're passing in the account details directly.

```suggestion
	setup()
```

Reply:
Not including `UsingAccount` leaves `api.AccountID` as the zero value (empty string) causing all of these tests to fail. Should I not be using `api.AccountID`?

Reply:
no, you shouldn't. instead the method should accept the account ID as a part of the method signature or parameters.

Reply:
there will be a lint rule coming for this soon but for now, i've just removed it.